sdk/spring/azure-spring-boot-bom/pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-spring-boot-bom</artifactId>
+    <version>1.0.0-beta.1</version>
+    <packaging>pom</packaging>
+
+    <name>Azure Spring Boot BOM</name>
+    <description>BOM for Microsoft Azure Spring Boot Support</description>
+    <url>https://github.com/Microsoft/azure-spring-boot</url>
+
+    <licenses>
+        <license>
+            <name>MIT</name>
+            <url>https://github.com/Microsoft/azure-spring-boot/blob/master/LICENSE</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <id>ZhijunZhao</id>
+            <name>Zhijun Zhao</name>
+            <email>zhijzhao@microsoft.com</email>
+        </developer>
+
+        <developer>
+            <id>yungez</id>
+            <name>Yunge Zhu</name>
+            <email>yungez@microsoft.com</email>
+        </developer>
+    </developers>
+
+    <organization>
+        <name>Microsoft</name>
+        <url>https://www.microsoft.com</url>
+    </organization>
+
+    <scm>
+        <connection>scm:git:git://github.com/Microsoft/azure-spring-boot-starters.git</connection>
+        <developerConnection>scm:git:ssh://github.com:Microsoft/azure-spring-boot-starters.git</developerConnection>
+        <url>https://github.com/Microsoft/azure-spring-boot/tree/master</url>
+    </scm>
+
+    <properties>
+        <azure.spring.boot.version>1.0.0-beta.1</azure.spring.boot.version>
+        <azure.dependencies.bom.version>2.2.0.M4</azure.dependencies.bom.version>
+        <spring.data.cosmosdb.version>2.2.3</spring.data.cosmosdb.version>
+        <spring.data.gremlin.version>2.2.2</spring.data.gremlin.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.microsoft.azure</groupId>
+                <artifactId>spring-data-cosmosdb</artifactId>
+                <version>${spring.data.cosmosdb.version}</version>
```

Comment:
Changed.

Reply:
/CC: @JonathanGiles @alzimmermsft 
If this is a creating a bom release, should this be in here or should this be another entry in the [sdk/boms](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/boms)?
Also, to @srnagar's comment about the versioning tags...Since BOMs aren't updated as frequently as the versions we're releasing and because of this we haven't been using the version tags in the pom files. I would rather see the version directly in the dependency instead of property strings.